DockerHubfile

```diff
@@ -1,32 +1,32 @@
-#### Create Jar 
-FROM gradle:jdk8 as builder 
- 
-COPY . /home/gradle/app 
+#### Create Jar
```

Comment:
By any chance did we check Docker hub if the new image file is created ?

Reply:
I did not. By the looks it should not have worked since the jar of project was not suffixed with `-api`